src/default-test/java/nl/tudelft/jpacman/npc/ghost/NavigationTest.java

```diff
@@ -43,8 +45,10 @@
     @BeforeEach
     void setUp() {
         PacManSprites sprites = new PacManSprites();
-        parser = new MapParser(new LevelFactory(sprites, new GhostFactory(
-            sprites)), new BoardFactory(sprites));
+        LevelFactory lfact = new LevelFactory(sprites,
```

Comment:
Maybe change `lfact` to `levelFactory` for clarity. `sprites` on the next line would make more sense imo.